web-services/deploy/docs/docs/query_help.html

```diff
@@ -209,5 +209,17 @@ <h3>LUCENE Query Syntax</h3>
 		</ul>
 		<li>A special date format of 'e' can be supplied to mean milliseconds since epoch.</li>
 	</ol>
+	<h3>Unique Functions</h3>
+	<p class="left">In addition, there are functions that are available to visit the query tree, and perform operations on the nodes and their
```

Comment:
I may be off base but here's what I'm thinking:

This document's audience will be end users. There is a "query lingo" we can assume knowledge of - like "index-only" fields or "events". But in this paragraph you're talking about the query parse tree and visitors - that's going to be out of their realm. This belongs more in a javadoc I think.

Reply:
Makes sense. So maybe replace the excerpt there with "Checks the fields to be unique over the interval of a day/minute/second", or something along those lines?

Reply:
Yeah, the wording is a bit tough. Might be best to snag a query SME for the phrasing but I'd guess something like "Returns one result per unique value in the given timeframe" or something like that?

Reply:
I've updated the description in the latest commit, and provided a few more contextual examples.